torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16053,6 +16058,12 @@ def error_inputs_mean(op_info, device, **kwargs):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            sample_inputs_func=sample_inputs_ravel,
+           skips=(
+               # the stride of the tensor was modified directly without going through the PyTorch dispatcher.
+               DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_backward'),
+               # the stride of the tensor was modified directly without going through the PyTorch dispatcher.
+               DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_forward_ad'),
+           )
```

Comment:
The tests failed after adding a new non-contig input

Reply:
Because noncontiguous creation in `make_tensor` relies on `index_select`

Reply:
Right, sorry, those particular tests are failing for other reasons

Reply:
Sorry. I see that I commented at wrong line.